test/Microsoft.Windows.CsWin32.Tests/TestUtils.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Windows.CsWin32.Tests
+{
+    using System.Text.RegularExpressions;
+
+    internal static class TestUtils
+    {
+        private const string ExpectedGeneratedSourceLineEnding = "\r\n";
+        private static readonly Regex NewLineRegex = new(@"\r?\n");
+
+        public static string NormalizeToExpectedLineEndings(string text)
+        {
+            return NewLineRegex.Replace(text, ExpectedGeneratedSourceLineEnding);
```

Comment:
line endings on multi-line string checks can be tough in git. If this works though, it's good enough for me.

Reply:
Yep, it does work.